anax-in-container/horizon-container

```diff
@@ -154,20 +154,21 @@ copyHostOSInfo() {
     fi
 
 	# write os, distro, distro version and architecture to file
-    echo "OS=$os" > host-os-release
-    echo "DISTRO=$distro" >> host-os-release
-    echo "DISTRO_VERSION_NUM=$distroVersionNum" >> host-os-release
-    echo "ARCH=$image_arch" >> host-os-release
+	tmpfile="/tmp/host-os-release-${DOCKER_NAME}"
+    echo "OS=$os" > $tmpfile
+    echo "DISTRO=$distro" >> $tmpfile
+    echo "DISTRO_VERSION_NUM=$distroVersionNum" >> $tmpfile
+    echo "ARCH=$image_arch" >> $tmpfile
 
 	# copy file to docker container
-    docker cp host-os-release $DOCKER_NAME:/etc/host-os-release
+    docker cp $tmpfile $DOCKER_NAME:/etc/host-os-release
 	if [[ $? -ne 0 ]]; then
 		echo "Could not copy host os info into agent docker container"
 		exit 2
 	fi
 
 	# cleanup
-    rm host-os-release
+    rm $tmpfile
 }
```

Comment:
Looks like there is some inconsistent indentation in this section

Reply:
fixed